test/unit/unix/csh.test.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @overview Contains (additional) unit tests for the escaping functionality for
+ * the C shell.
+ * @license Unlicense
+ */
+
+import { testProp } from "@fast-check/ava";
+import * as fc from "fast-check";
+import utf8 from "utf8";
```

Comment:
Following up on the resolution to [this earlier comment](https://github.com/ericcornelissen/shescape/pull/659#discussion_r1080663574), let's try to remove the use of the [`utf8` library](https://www.npmjs.com/package/utf8) here as well.

Reply:
Achieved using [the `TextDecoder` class](https://nodejs.org/api/util.html#class-utiltextdecoder) in e95f2243daf88f960c4066456b0a8a5ccb320d4c.